org.trustedqsl.Tqsl.yml

```diff
@@ -0,0 +1,63 @@
+app-id: org.trustedqsl.Tqsl
+runtime: org.kde.Platform
+runtime-version: '5.15'
+sdk: org.kde.Sdk
+command: tqsl
+rename-desktop-file: tqsl.desktop
+rename-icon: tqsl
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --device=all
+  - --share=network
+  - --persist=.tqsl
```

Comment:
I think that --persist does not work in combination with --filesystem=home. If --filesystem=home is enabled, the config directory is always created in ~/ instead of the application directory in ~/.var/app. You can probably remove the --persist line.